openupgrade_scripts/scripts/sale_stock/16.0.1.0/pre-migration.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2023 Coop IT Easy SC
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+def compute_sale_order_delivery_status(env):
+    openupgrade.add_fields(
+        env,
+        [
+            (
+                "delivery_status",
+                "sale.order",
+                False,
+                "selection",
+                False,
+                "sale_stock",
+            )
+        ],
+    )
```

Comment:
```suggestion
    openupgrade.add_fields(
        env,
        [
            (
                "delivery_status",
                "sale.order",
                 "selection",
                "character varying",
                "sale_stock",
                False,
            )
        ],
    )
```

@huguesdk  I think the order should be: field name, model name, SQL table name, field type, SQL field type, module name, (optional) initialization value
Maybe i'm wrong, can you check it again

Reply:
[here](https://github.com/OCA/openupgradelib/blob/master/openupgradelib/openupgrade.py#L2816) is the definition of `openupgrade.add_fields()`. the way the arguments work are not that intuitive. the only mandatory elements are the field name (0), the model name (1), the field type (3) and the module name (5). if the others are false, they will be determined automatically. i think the code is correct as it is. with your suggestion, it would try to add a column of type `sale_stock` to the `selection` table, for the `False` module. 🤔

Reply:
@huguesdk ok bro :+1: